assets/js/admin/admin.js

```diff
@@ -300,6 +300,54 @@ jQuery(function ($) {
 	} else {
 		minimum_password_strength_wrapper_field.hide();
 	}
+	var password_strength_option = minimum_password_strength_wrapper_field.find(
+		"#user_registration_form_setting_minimum_password_strength"
+	);
+
+	// show password strength info.
+	$(document).ready(function () {
+		var strength_info = "";
+		var password_hint = "";
+		var password_strength_value = password_strength_option
+			.find(":selected")
+			.val();
+		show_password_strength_info(password_strength_value);
+
+		$(password_strength_option).on("change", function () {
+			password_hint =
+				minimum_password_strength_wrapper_field.find("span");
+			$strength = $(this).find(":selected").val();
+			password_hint.remove();
+			show_password_strength_info($strength);
+		});
+		function show_password_strength_info($strength_value) {
+			switch ($strength_value) {
+				case "0":
+					strength_info = "Minimum one uppercase letter";
```

Comment:
@dipuchaudhary Please make text translatable and localize it

Reply:
Done by me.